### DISCUSSION TITLE
The abstract level (granularity of functions) in the functional architecture

### DISCUSSION THREAD

**Discussion by PrinceAlliander** [lfenergyarchitecturemodel/lfenergyfunctionalarchitecturemodel#85] [General]
At what level of abstraction do we want to model functions in the function architecture. I propose we started with refining the functions which we already have modeled and converted/rename the functions or objects which are at the application/technology or physical layer. Comments welcome ...

**Comment by Jonasvdbo** [comment]
As discussed in the LF Energy Architecture model TSC meeting, we will focus on the business functions.
